chapters/equations.tex

```diff
@@ -264,6 +269,8 @@ \subsubsection{Restrictions on Equations within When-Equations}\label{restrictio
 \end{lstlisting}
 \end{example}
 
+\paragraph*{Restrictions on equations within when-equations:}\label{restrictions-on-equations-within-when-equations}
```

Comment:
Possible to merge this with the sentence below that introduces the list?

Reply:
On second thoughts, I think using `\subsubsection` would be better.

Also note that attaching a label to a starred sectioning command looks like a source of confusion.

Reply:
I agree that subsubsection becomes better, and it just lifts these two lists one level.